lib/eval.coffee

```diff
@@ -34,6 +34,31 @@ module.exports =
           @showError r, result.highlights
         notifications.show "Evaluation Finished"
 
+  # get documentation or methods for the current word
+  toggleMeta: (type) ->
+    editor = atom.workspace.getActiveTextEditor()
+    [word, range] = @getWord editor
+    # if we only find numbers or nothing, return prematurely
+    if word.length == 0 || !isNaN(word) then return
+    client.msg type, {code: word}, ({result}) =>
+      view = if result.type then result.view else result
+      view = @ink.tree.fromJson(view)[0]
+      @ink.links.linkify view
+      r = @ink?.results.toggleUnderline editor, range,
+        content: view
+        clas: 'julia'
+
+  # gets the word and its range in the `editor` which the last cursor is on
+  getWord: (editor) ->
+    cursor = editor.getLastCursor()
+    # The following line is kinda iffy: The regex may or may not be well chosen
+    # and it duplicates the efforts from atom-language-julia. It might be
+    # better to select the current word via finding the smallest <span> containing
+    # the cursor which also has `function` or `macro` as its class.
+    range = cursor.getCurrentWordBufferRange({wordRegex: /[\w_!´]*\.?@?[\w_!´]+/})
```

Comment:
I'm at this regex now: `/[\u00A0-\uFFFF\w_!´]*\.?@?[\u00A0-\uFFFF\w_!´]+/` (per the link you gave me). It does seem to work, but I'm pretty sure there's lots of stuff in that char-range that isn't valid Julia.

I'll have a look at Julia's parser and see if there's an easy way of extracting valid charcodes...

As for changing the grammar: Is it possible to have nested scopes (haven't looked at it much so far)? Ideally we'd have something like 
```HTML
<span class="function"><span class="module">Base</span>.<span class="method">sin</span></span>
```
or similar so that it's easy to extract the whole function call.